packages/mobile/src/exchange/ExchangeReview.tsx

```diff
@@ -113,6 +116,7 @@ export class ExchangeReview extends React.Component<Props, State> {
       makerAmount = getTakerAmount(inputAmount, exchangeRate)
     }
     this.props.fetchExchangeRate(makerToken, makerAmount)
+    this.props.fetchTobinTax(makerAmount, makerToken)
```

Comment:
I think my old comment about this got buried so posting again:

In the time btwn when this is dispatched, and when the value comes in, the old value will still live in the store and be shown in the UI right? Does the UI block until that's in? Or can a user proceed with the incorrect value?
Ditto concern for fetchExchangeRate above. We may need a solution like what we do for fees, where things block while those are refreshed.

And something else I just noticed, a `get` method like this should be pure. It's a bad place to dispatch these actions (fetch tobin tax and fetch exchange rate). These should be in a componentDidMount of DidUpdate

Reply:
Updated to dispatch the actions in componentDidMount- had originally written it this way to avoid having to pass makerToken/makerAmount within componentDidMount, but agree that it's unexpected to dispatch an action in this getter

Reply:
Created an issue to track the UI blocking- https://github.com/celo-org/celo-monorepo/issues/2472